.env.development

```diff
@@ -2,7 +2,10 @@ BASE_URL='http://localhost:8734'
 CATALOG_BASE_URL='http://localhost:18000/courses'
 LMS_BASE_URL='http://localhost:18000'
 ECOMMERCE_BASE_URL='http://localhost:18130'
+DISCOVERY_API_URL='http://localhost:18381/api'
+ENTERPRISE_CATALOG_API_URL='http://localhost:18160/api'
 ORDER_HISTORY_URL='http://localhost:1996/orders'
+MARKETING_SITE_URL='https://stage.edx.org'
```

Comment:
I'm using stage for the `MARKETING_SITE_URL` as I don't have a local copy of Prospectus running for linking to pages that live there (e.g., subject pages, instructor bios, etc.).